submission.md

```diff
@@ -66,3 +66,95 @@ The final Item's UUID will be the same as it was in the WorkspaceItem (i.e. the
 `/api/submission/workspaceitems/<:id>/item`)
 * If the Collection has an approval workflow configured, then a WorkflowItem will be returned. Its `id` can be used
 to access the WorkflowItem via `/api/workflow/workflowitems/<:id>`.
+
+## Finding potential duplicate items
+
+**GET /api/submission/duplicates/search?uuid=<:uuid>**
```

Comment:
I'd recommend moving this documentation to a new `duplicates.md` section.  This `submission.md` page is an **overview of how to perform a submission** from the REST API.  So, this doesn't belong here, as it's not a task you need to do to perform a submission from the UI.

Instead, this should have it's own "page" in the REST Contract, just like how the `/api/submission/vocabularies` endpoint (for controlled vocabs) has it's own page in `vocabularies.md`.

Reply:
No problem, done. also linked from endpoints.md